src/cli/telemetry/shouldTrack.ts

```diff
@@ -0,0 +1,11 @@
+import { isInteractive } from './helpers';
+import { checkTelemetry } from './telemetry';
+
+/**
+ * Used to determine if tracking should occur.
+ * @param ci boolean
+ * @returns boolean
```

Comment:
IMO, we can get the type information from the function header here.  I would suggest being a little more descriptive here:
```suggestion
 * @param ci whether or not the process is running in a Continuous Integration (CI) environment
 * @returns true if telemetry should be sent, false otherwise
```